mubench.pipeline/tasks/implementations/check.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import List
+
+from tasks.project_task import Requirement
+from tasks.project_task import ProjectTask
+from utils.shell import Shell
+
+
+class Check(ProjectTask):
+    def __init__(self):
+        super().__init__()
+
+    def start(self):
+        self._check_all_requirements()
+
+    def process_project(self, *_):
+        return self.ok()
+
+    def _check_all_requirements(self):
+        logger = logging.getLogger("requirements")
+        logger.info("Checking all requirements...")
```

Comment:
I'm not 100% sure, but the Task Runner should already output something like "Running Check...", so we should drop this reduce output. I would suggest we rename the task to `RequirementsCheck`, to make the output more expressive.